src/WebJobs.Script/Description/Node/Script/scriptRequire.js

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I was planning on making all the logic currently in functionTemplate.js an actual module that exports it's various functions, and eventually adding Mocha tests for them. That's been a TODO.

Since you're half way there now, I'm wondering if you shouldn't just create the module now, do a `var functions = require('azure-functions')` in this file, and simply `return functions.create(f)`. You get the idea. That will facilitate us adding Mocha tests in the future.